.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "sourcemod_plugins/get5"]
+	path = sourcemod_plugins/get5
+	url = https://github.com/splewis/get5.git
```

Comment:
Would it make more sense to only add the `get5_eventapi` submodule and add `get5` as a submodule of `get5_eventapi`?

Reply:
I don't think so because I forked only `get_apistats` from `get5` while for the discord 10 man bot project, it also needs `get5` as a sourcemod plugin.